MinnPost/object-sync-for-salesforce
# Upgrade ActionScheduler library to version 3.2

## What does this Pull Request do?
When it is merged, this pull request will close #299 and upgrade our installation of the ActionScheduler library to version 3.2. This will start several processes that the library runs to upgrade itself. Read about that process [on their site](https://actionscheduler.org/version3-0/).

## How do I test this Pull Request?

- take a database backup of your site
- enable [WordPress debugging](https://wordpress.org/support/article/debugging-in-wordpress/) if it isn't already enabled
- go to the `/wp-admin/tools.php?page=action-scheduler` URL in your WordPress install after checking out this branch
- basically just watch what it says, and compare with what the documentation link above says
- if you have database access, make sure it creates and popuulates the new tables: `wp_actionscheduler_actions`, `wp_actionscheduler_claims`, `wp_actionscheduler_groups`, and `wp_actionscheduler_logs` (current as of 5/19/21).
- check your PHP error logs for anything relevant
